apps/bwclk/app.js

```diff
@@ -370,7 +370,8 @@ Bangle.loadWidgets();
 
 // Cache draw function for dynamic screen to hide / show widgets
 // Bangle.loadWidgets() could also be called later on but its much slower!
-for (let wd of WIDGETS) {wd._draw=wd.draw; wd._area=wd.area;}
+//for (let wd of WIDGETS) {wd._draw=wd.draw; wd._area=wd.area;} // Not sure how to replace this or if it should just be dropped?
+widget_utils.show(); // I replace with this to begin with, should I just delete it?
```

Comment:
// Not sure how to replace this or if it should just be dropped?

widget_utils.show(); // I replace with this to begin with, should I just delete it?